### ISSUE TITLE
128 sample in TextbookQA

### ISSUE THREAD

**Issue by yinzhangyue** [oriram/splinter#10]
Hello, I have some problem when evaluate in TextbookQA. The result is frustrated. I use the Splinter from your dropbox. Here is my eval_result and finetune_splinter.sh.
```
Final Eval:
exact = 23.750
**f1 = 29.341**
total = 400
HasAns_exact = 23.750
HasAns_f1 = 29.341
HasAns_total = 400
best_exact = 23.750
best_exact_thresh = 0.000
best_f1 = 29.341
best_f1_thresh = 0.000
```

```
export MODEL="../../splinter"
export dataset="textbookqa"
export OUTPUT_DIR="../../output_model/${dataset}_128"
python ../run_mrqa.py \
    --model_type=bert \
    --model_name_or_path=$MODEL \
    --qass_head=True \
    --tokenizer_name=$MODEL \
    --output_dir=$OUTPUT_DIR \
    --train_file="../../mrqa-few-shot/${dataset}/${dataset}-train-seed-42-num-examples-128.jsonl" \
    --predict_file="../../mrqa-few-shot/${dataset}/dev.jsonl" \
    --do_train \
    --do_eval \
    --max_seq_length=512 \
    --doc_stride=128 \
    --threads=4 \
    --save_steps=50000 \
    --per_gpu_train_batch_size=16 \
    --per_gpu_eval_batch_size=16 \
    --learning_rate=3e-5 \
    --max_answer_length=10 \
    --warmup_ratio=0.1 \
    --min_steps=200 \
    --num_train_epochs=32 \
    --seed=42 \
    --use_cache=False \
    --evaluate_every_epoch=True  \
    --overwrite_output_dir
```

**Comment by oriram** [comment]
Hi @yinzhangyue, did you try running with the other datasets (i.e. seed-43 to seed-46)?
If so, from a quick glance I can see some differences in HPs which might effect results:
**Batch size**: you set it to 16 while we use 12,
**max_seq_length**: you set it to 512 while we use 384.

**Comment by yinzhangyue** [comment]
I reproduce your result on seed-43. I also find the seed is very important and the variance on different seed is a little big. Do you have the same situation in your experiment?

**Comment by oriram** [comment]
The standard variation of all experiments is reported in the appendix (Table 3).
For TBQA with 128 examples, the result is 42.6 (std of 2.5). 
Generally, the std is larger as fewer examples are considered.

**Comment by yinzhangyue** [comment]
OK. I see it. I mean have you ever had such frustrating results in your experiments (29.341 is so terrible)
